third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -752,4 +753,58 @@ void dds_device::impl::on_stream_options( json const & j, dds_sample const & sam
 }
 
 
+void dds_device::impl::on_calibration_changed( json const & j, dds_sample const & sample )
+{
+    for( auto const & name_stream : _streams )
+    {
+        auto & stream = name_stream.second;
+
+        auto j_int = j.nested( stream->name(), topics::notification::calibration_changed::key::intrinsics );
+        if( ! j_int )
+            return;  // stream isn't updated
```

Comment:
Continue to next stream?
Some calibrations may update intrinsics of more than one stream.

Reply:
Right - I think this was in a lambda before, hence the return

Reply:
Fixed